package/package_scripts_dist_cloudReader.cson

```diff
@@ -66,7 +68,7 @@
     '
 
 #=====================
-    'dist:cloudReader:css': '
+'dist:cloudReader:css': '
 
```

Comment:
@derekriemer the 4-spaces indentation before the `dist:cloudReader:css` task-name must be preserved (it was removed in your commit). Otherwise, the build command `npm run cson2json` breaks.